tools/explorer/pkg/stellar/stellar.go

```diff
@@ -144,9 +151,111 @@ func (w *Wallet) CreateAccount() (keypair.Full, error) {
 		return keypair.Full{}, errors.Wrap(hError.Problem, "error submitting transaction")
 	}
 
+	if len(w.signers) == 5 {
```

Comment:
So we only create the mutlisign transaction if we have 5 signer address define. Shouldn't we allow creating the transaction as soon as we have 3 or more signers ? Since we requires only 3 to be able to create transaction later on ?

Reply:
And shouldn't this all block be move into it's own function ?

Reply:
Well we said we were going to add 5 signers to the escrow accounts which of only 3 out of 5 must sign off on a transaction.